kpi/schema_extensions/v2/assets/extensions.py

```diff
@@ -1,10 +1,583 @@
+    target_class = 'kpi.schema_extensions.v2.assets.fields.CountDailySubmissionResponseField'  # noqa
+
+    def map_serializer_field(self, auto_schema, direction):
+        return build_object_type(
+            properties={
+                '2020-10-20': build_basic_type(OpenApiTypes.INT),
+            }
+        )
+
+
+class DataFieldExtension(OpenApiSerializerFieldExtension):
+    target_class = 'kpi.schema_extensions.v2.assets.fields.DataURLField'
+
+    def map_serializer_field(self, auto_schema, direction):
+        return build_url_type(
+            'api_v2:submission-list',
+            parent_lookup_asset='aBeA23YCYjkGTFvYVHuAyU',
+        )
+
+
+class DataSharingFieldExtension(OpenApiSerializerFieldExtension):
+    target_class = 'kpi.schema_extensions.v2.assets.fields.DataSharingField'
+
+    def map_serializer_field(self, auto_schema, direction):
+        return build_object_type(properties={})
+
+
+class DeploymentActiveFieldExtension(OpenApiSerializerFieldExtension):
+    target_class = 'kpi.schema_extensions.v2.assets.fields.DeploymentActiveField'
+
+    def map_serializer_field(self, auto_schema, direction):
+        return build_basic_type(OpenApiTypes.BOOL)
+
+
+class DeploymentDataDownloadLinksFieldExtension(OpenApiSerializerFieldExtension):
+    target_class = (
+        'kpi.schema_extensions.v2.assets.fields.DeploymentDataDownloadLinksField'
+    )
+
+    def map_serializer_field(self, auto_schema, direction):
+        return build_object_type(properties={})
+
+
+class DeploymentLinkFieldExtension(OpenApiSerializerFieldExtension):
+    target_class = 'kpi.schema_extensions.v2.assets.fields.DeploymentLinkField'
+
+    def map_serializer_field(self, auto_schema, direction):
+        return build_object_type(properties={})
+
+
+class DeploymentSubmissionCountFieldExtension(OpenApiSerializerFieldExtension):
+    target_class = (
+        'kpi.schema_extensions.v2.assets.fields.DeploymentSubmissionCountField'
+    )
+
+    def map_serializer_field(self, auto_schema, direction):
+        return build_basic_type(OpenApiTypes.INT)
+
+
+class DeployedVersionFieldExtension(OpenApiSerializerFieldExtension):
+    target_class = 'kpi.schema_extensions.v2.assets.fields.DeployedVersionField'
+
+    def map_serializer_field(self, auto_schema, direction):
+        return build_object_type(
+            properties={
+                'count': build_basic_type(OpenApiTypes.INT),
+                'next': build_basic_type(OpenApiTypes.STR),
+                'previous': build_basic_type(OpenApiTypes.STR),
+                'results': build_array_type(
+                    schema=build_object_type(
+                        properties={
+                            'uid': build_basic_type(OpenApiTypes.STR),
+                            'url': build_url_type(
+                                'asset-detail',
+                                uid='aBeA23YCYjkGTFvYVHuAyU',
+                            ),
```

Comment:
Nope! The url should be for versions, i.e. `asset-version-detail` with an uid which starts with `v`.